src/libsyncengine/propagation/executor/executorworker.cpp

```diff
@@ -1654,11 +1644,15 @@ bool ExecutorWorker::handleFinishedJob(std::shared_ptr<AbstractJob> job, SyncOpP
         NodeId locaNodeId;
         NodeId remoteNodeId;
         if (syncOp->targetSide() == ReplicaSideLocal) {
-            locaNodeId = syncOp->correspondingNode() && syncOp->correspondingNode()->id().has_value() ? syncOp->correspondingNode()->id().value() : std::string();
+            locaNodeId = syncOp->correspondingNode() && syncOp->correspondingNode()->id().has_value()
+                             ? syncOp->correspondingNode()->id().value()
```

Comment:
You can use safely use `*(syncOp->correspondingNode()->id())` instead of `syncOp->correspondingNode()->id().value()` since we check the existence of the value beforehand. This is slightly short and faster.

Reply:
Since `localNodeId` is initialized with `std::string{}`, you can replace the ternary operator with a conditional assignment: `if(<condition>) { <assign>; };`

Reply:
Done, thanks